PPE[ Lab-5 ]/PPE[ Lab-5 ]/Form1.cs

```diff
@@ -119,7 +120,7 @@ private void Timer_Tick(object sender, EventArgs e)
                 this.ScoreLabel.Text = "Score : " + score;
 
 				// increase speed when snake eats 2 (apples)
```

Comment:
update comment or remove completely (as you seem to % by 5, but the comment is confusing)